.github/workflows/golangci_lint.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         with:
```

Comment:
Please also bring the CI action from here: https://github.com/smartcontractkit/chainlink-aptos/pull/75#discussion_r2032753414

Reply:
I added it in but it is currently failing since there are no protobuf definitions in the main branch. But it should work after this is merged to main